### PR TITLE
chore(release): bump version to 2.0.0-a1

### PR DESCRIPTION
**Pull request by kulgan**

### CODE REVIEW

**Comment by kulgan** [comment]
> I guess we can revisit version numbers when we start pushing things to Nexus.
> 
> Do we also need to update the `dependency_links` in `setup.py`?

I was going to remove them entirely, but forgot. We don't really need them to do travis

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 future
 SQLAlchemy==1.3.3
--e git+https://github.com/NCI-GDC/psqlgraph.git@release/py3#egg=psqlgraph
--e git+https://github.com/NCI-GDC/gdcdictionary.git@release/py3#egg=gdcdictionary
--e git+https://github.com/NCI-GDC/gdc-ng-models.git@release/redfish#egg=gdc_ng_models
+git+https://github.com/NCI-GDC/psqlgraph.git@2.0.0-alpha#egg=psqlgraph
+git+https://github.com/NCI-GDC/gdcdictionary.git@2.0.0-alpha#egg=gdcdictionary
+git+https://github.com/NCI-GDC/gdc-ng-models.git@1.1.0#egg=gdc_ng_models
```
Comment (philtom-ctds) [inline requirements.txt]: This version changed from "release/redfish" to "1.1.0".  How would I know this is the correct version for a given release?

Comment (kulgan) [reply requirements.txt]: `redfish` was `1.1.0` for that repo. You can check the releases on the repo url

Comment (philtom-ctds) [reply requirements.txt]: Do we have a convention for when to use a release name vs a version?

**Pull request merged**